app/src/main/kotlin/org/akanework/gramophone/ui/fragments/DetailDialogFragment.kt

```diff
@@ -82,6 +84,8 @@ class DetailDialogFragment : BaseFragment(false) {
         mimeTypeTextView.text = mediaItem.localConfiguration?.mimeType ?: "(null)"
         pathTextView.text = mediaItem.getFile()?.path
             ?: mediaItem.requestMetadata.mediaUri?.toString() ?: "(null)"
+        val bitrate = mediaItem.getBitrate(requireContext())
+        bitRateTextView.text = if (bitrate > 0) "${bitrate / 1000} kbps" else "(no bitrate)"
```

Comment:
Please make both the kbps string (Android supports format strings) and "(no bitrate)" translatable, and maybe consider saying "unknown" instead of "no"